src/axom/sina/docs/sphinx/records.rst

```diff
@@ -0,0 +1,428 @@
+.. ## Copyright (c) 2017-2024, Lawrence Livermore National Security, LLC and
+.. ## other Axom Project Developers. See the top-level LICENSE file for details.
+.. ##
+.. ## SPDX-License-Identifier: (BSD-3-Clause)
+
+.. _records-label:
+
+=======
+Records
+=======
+
+Sina ``Record`` objects are used to represent the data to be stored for your
+study. Some examples for the natural scope of ``Record`` objects include things
+like:
+
+    - a single run of an application
+    - an msub job
```

Comment:
I think msub has been gone for a while now. Does this mean to say job launch parameters would be good thing to store as a record?

Reply:
I copied a good amount of these docs from what Sina already had existing so this was probably put in there a while back. @doutriaux1 @HaluskaR I'm going to defer to you on this as I have no experience with msub and can't say for certain what was meant by this